pom.xml

```diff
@@ -89,6 +89,14 @@
             </ignores>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
```

Comment:
By the way, for official plugins this can be omitted if you like:
```suggestion
```
or kept for legibility.